precompile/bank/IBank.sol

```diff
@@ -12,8 +12,8 @@ interface IBank {
     function send(
         address fromAddress,
         address toAddress,
-        string calldata denom,
-        uint256 amount
+        string[] calldata denom,
+        uint256[] calldata amount
```

Comment:
I think it would be better to declare a `Coin` type and have it in the Coin[] format.

Reply:
[a68f7cd](https://github.com/xpladev/xpla/pull/166/commits/a68f7cde52fe3ce7109ebf87bb29b65269c03095)